src/client/screen/FeedScreen/index.js

```diff
@@ -0,0 +1,136 @@
+import VideosFull from '../../components/VideoFullViewComponent';
+import './index.scss';
+ 
+ class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'videoLists': [],
+      'clickUpload': false,
+      'errorMessage': null,
+      'uploading': false,
+      'loginuser': {},
+    };
+    this.onClickUpload = this.onClickUpload.bind(this);
+    this.onClickCancelUpload = this.onClickCancelUpload.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+  componentDidMount() {
+    this.fetchVideoLists((result) => {
+      if (result.error) {
+        return;
+      }
+      this.setState({videoLists: result});
+    });
+    this.fetchLoginUserInfos((result) => {
+      if (result.error) {
+        return;
+      }
+      this.setState({loginuser: result});
+    });
+  }
+  fetchLoginUserInfos(callback) {
+    fetch('/api/loginuser', {headers: {'Authorization': localStorage.getItem('token')}})
+      .then((response) => response.json())
+      .then((result) => callback(result));
+  }
+  fetchVideoLists(callback) {
+    fetch('/api/videos')
+      .then((response) => response.json())
+      .then((result) => callback(result));
+  }
+  onClickUpload() {
+    this.setState({clickUpload: true});
+  }
+  onClickCancelUpload() {
+    this.setState({
+      clickUpload: false,
+      errorMessage: null,
+      uploading: false,
+    });
+  }
+  onSubmit(ev) {
+    ev.preventDefault();
+    let statusCode;
+    let obj = {
+      url: ev.target.elements.namedItem('video-url').value || undefined,
+      preview: ev.target.elements.namedItem('video-preview').value || undefined,
+      title: ev.target.elements.namedItem('video-title').value || undefined,
+    };
+
+    this.setState({uploading: true});
+    fetch('/api/videos', {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': localStorage.getItem('token'),
+      },
+      body: JSON.stringify(obj),
+    }).then((response) => {
+      statusCode = response.status;
+      return response.json();
+    }).then((result) => {
+      if (statusCode === 201) {
+        this.fetchVideoLists((newVideoLists) => {
+          if (newVideoLists.error) {
+            return;
+          }
+          this.setState({videoLists: newVideoLists});
+        });
+        this.onClickCancelUpload();
+      } else {
+        this.setState({
+          errorMessage: result.error,
+          uploading: false,
+        });
+      }
+    });
+  }
+  render() {
+    return (
+      <div className="homecontainer">
+        <Header className="header"
+          onClickUpload={this.onClickUpload}
+          userInfos={this.state.loginuser}
+        />
+        <div className="main">
+          {this.state.clickUpload ?
+            <form className="upload-form" onSubmit={this.onSubmit}>
+              <input type="text" name="video-url" placeholder="video url" required
```

Comment:
use the [type="url"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url) attribute instead